jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -112,7 +112,7 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ecma_string_t *number_str_p = ecma_get_string_from_value (string_var);
   lit_utf8_size_t str_size = ecma_string_get_size (number_str_p);
 
-  MEM_DEFINE_LOCAL_ARRAY (utf8_string_buff, str_size, lit_utf8_byte_t);
+  MEM_DEFINE_LOCAL_ARRAY (utf8_string_buff, str_size + 1, lit_utf8_byte_t);
```

Comment:
@galpeter, as string is empty in the test case from #428, maybe we don't need to process it. So, instead of `+1` we could put the code into `if` block with fastpath for empty string.